internal/command/kind.go

```diff
@@ -21,14 +21,24 @@ import (
 	"knative.dev/kn-plugin-quickstart/pkg/kind"
 )
 
+var kindClusterName string
+
 // NewKindCommand implements 'kn quickstart kind' command
 func NewKindCommand() *cobra.Command {
-	return &cobra.Command{
+	var kindCmd = &cobra.Command{
 		Use:   "kind",
 		Short: "Quickstart with Kind",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			fmt.Println("Running Knative Quickstart using Kind")
-			return kind.SetUp()
+			return kind.SetUp(kindClusterName)
 		},
 	}
+	// Set kindCmd options
+	kindOptions(kindCmd)
+
+	return kindCmd
+}
+
+func kindOptions(targetCmd *cobra.Command) {
+	targetCmd.Flags().StringVarP(&kindClusterName, "name", "n", "", "kind cluster name to be used by kn-quickstart, config (default kind)")
```

Comment:
Would it be better to set the default value to `kind` rather than `""`?

Reply:
Also, wonder if we should omit the `config` from the help message (i.e. `kind cluster name to be used by kn-quickstart (default kind)`